python/ray/worker.py

```diff
@@ -2286,6 +2298,7 @@ def _try_to_compute_deterministic_class_id(cls, depth=5):
 
 
 def register_custom_serializer(cls,
+                               driver_hash,
```

Comment:
Let's remove `driver_hash` from the arguments. `register_custom_serializer` is part of the API, and I this seems to be an unnecessary API change since we can just use `worker.trask_driver_id`, right?

Reply:
We need it, since on the remote side we call this function in [this way](https://github.com/ray-project/ray/blob/8a3e180156c4d1fd2d611f464231f25ba7b3e08b/python/ray/import_thread.py#L173), that means the worker instance on remote side is passed in. Then worker.trask_driver_id cannot indicate the correct driver that the current class related to.

Reply:
I knew that. See the link I posted above. It is register_class_for_serialization that being called on remote side, but the parameter (worker) passed in on remote side is the remote worker.